galata/test/jupyterlab/settings.test.ts

```diff
@@ -31,9 +31,19 @@ test('Open the settings editor with a specific search query', async ({
 
   const settingsPanel = page.locator('.jp-SettingsPanel');
 
-  expect(await settingsPanel.screenshot()).toMatchSnapshot(
-    'settings-panel.png'
-  );
+  expect
+    .soft(await settingsPanel.screenshot())
+    .toMatchSnapshot('settings-panel.png');
+  // Test that new query takes effect
+  await expect(page.locator('.jp-PluginList-entry')).toHaveCount(0);
+
+  await page.evaluate(async () => {
+    await window.jupyterapp.commands.execute('settingeditor:open', {
+      query: 'CodeMirror'
+    });
+  });
+
+  await expect(page.locator('.jp-PluginList-entry')).toHaveCount(1);
```

Comment:
I think we need to change it to actually check that the entry is CodeMirror, not that it has count 1, because even before calling `execute('settingeditor:open', {query: 'CodeMirror'});` there is one entry, so this test would already pass on `main`